qiskit_experiments/library/tomography/tomography_experiment.py

```diff
@@ -88,6 +88,13 @@ def __init__(
             preparation_qubits: DEPRECATED, equivalent to preparation_indices.
             basis_indices: Optional, the basis elements to be measured. If None
                 All basis elements will be measured.
+            conditional_measurement_indices: Optional, a subset of `measurement_indices`
```

Comment:
Its definitely a bit confusing, and probably needs examples since this came from quite complicated research projects on circuit cutting and measurement tomography etc.

Essentially this is referring to the tomographic measurements added to a circuit by the experiment, ie to a subset the measurement_indices, which are to be used for performing conditional reconstruction. For standard basis this means measure in the Z-basis on these qubits, rather than the full basis, and this forces your reconstructed state to be block diagonal in this basis on those qubits. The other kwarg is used for conditioning on clbits already in a circuit (eg the circuit already contains measurements).

The main difference between these two cases is in the later we can't assume anything about what set the bit values (ie that it came from Z-basis measurement), since it could have happened anywhere in the circuit, while the former we know it was the outcome of a specific tomographic measurement what could have been used to reconstruct some larger state, and this information could be used for example to do measurement error mitigated conditional reconstruction (which is what we explored in https://arxiv.org/abs/2211.13431)

Reply:
> and this forces your reconstructed state to be block diagonal in this basis on those qubits

I like this description. This explains well what is _conditional_measurement_indices_. So basically you can reduce circuit number by specifying these indices?